docs/src/inputs/textArea/textArea.jsx

```diff
@@ -485,18 +485,19 @@ export default class ElementsTextArea extends React.Component {
             <Main page="headers">
                 <Main.Content>
                     <Card>
-                        <Header size="large">Props</Header>
+                        <Typography size="large">Props</Typography>
 
                         <TableProps props={props} />
                     </Card>
 
                     {/* Text Area */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub >
+                    <Typography size="large" style={{ marginTop: '55px' }} sub >
```

Comment:
Same ... let's nix the `sub`s if possible.  (Sounds like I'm suggesting anti-submarine warfare ;-)

Reply:
Fixed.

Reply:
`sub`s all seem to have been eliminated.  Resolving.